docs/topics/db/optimization.txt

```diff
@@ -261,44 +261,50 @@ But:
 
 .. _overuse_of_count_and_exists:
 
-Don't overuse ``count()`` and ``exists()``
-------------------------------------------
+Don't overuse ``count()``, ``exists()``, and ``contains()``
+-----------------------------------------------------------
 
 If you are going to need other data from the QuerySet, evaluate it immediately.
 
-For example, assuming an Email model that has a ``subject`` attribute and a
-many-to-many relation to User, the following code is optimal::
+For example, assuming a ``Group`` model that has a
+many-to-many relation to ``User``, the following code is optimal::
+
+    if display_group_members:
+        members = group.members.all()
+        if members:
+            if current_user in members:
+                print(f'You and {len(members) - 1} other users are members of this group')
```

Comment:
f-strings should use only plain variable and property access as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style). I've changed to the previous format.